themes/default/content/docs/intro/concepts/config.md

```diff
@@ -284,7 +284,20 @@ public static void stack(Context ctx) {
 
 {{< /chooser >}}
 
-## Special Configuration Options
+## Provider Configuration Options
+
+There are three ways to configure providers:
+
+1. Set configuration keys in the stack configuration file: `pulumi config set [PROVIDER]:[KEY] [VALUE]`
```

Comment:
Throughout the rest of this document - this is referred to as "Pulumi configuration" but here we use the term "stack configuration file". Suggest we continue to use "Pulumi configuration" in this section for consistency.